### PR TITLE
fix error sample manifests in readme.

### PR DESCRIPTION
**Pull request by morvencao**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -237,9 +237,8 @@ profile. Here's an example of overriding some global level default values:
 ```yaml
 profile: sds
 values:
-  global:
-    logging:
-      level: "default:warning" # override from info
+  logging:
+    level: "default:warning" # override from info
```
Comment (richardwxn) [inline README.md]: I think this part was correct before?

Comment (ostromart) [reply README.md]: Yes, you're right. Thanks.

Comment (ostromart) [reply README.md]: I mean that Morven is right i.e. there should be no global.

**Pull request merged**